scripts/merge.cr

```diff
@@ -64,11 +64,15 @@ end
 
 all_sponsors.sort_by! { |s| {-s.last_payment, -s.all_time, s.since, s.name} }
 
-File.open("#{__DIR__}/../_data/sponsors.csv", "w") do |file|
-  CSV.build(file) do |csv|
+write_csv("sponsors.csv", all_sponsors.select(&.listed?))
+write_csv("sponsor_logos_l.csv", all_sponsors.select { |sponsor| sponsor.last_payment.to_i > 1000 })
+write_csv("sponsor_logos_s.csv", all_sponsors.select { |sponsor| sponsor.last_payment.to_i.in?(350...1000) })
```

Comment:
Actually, I think we should leave the old entry threshold for at least the rest of this month to keep the logo of PlaceOS around because they have already paid for it.

```suggestion
write_csv("sponsor_logos_s.csv", all_sponsors.select { |sponsor| sponsor.last_payment.to_i.in?(250...750) })
```